### DISCUSSION TITLE
Formatting calendar entries

### DISCUSSION THREAD

**Discussion by ykharitonova** [kevinlin1/just-the-class#22] [Q&A] [answered]
Hello!

I'm working on [the calendar page](https://ucsb-csw8.github.io/m22-ykk/calendar/#week-2) for my course.

My questions are below; any help and guidance are much appreciated.

---

1) **Where can I adjust the width/alignment of the date field and the rightmost field?**

For example, I want to make the day of the week and the date appear on the same line by increasing the column width and/or the text alignment.
```
Mon, Aug 8:
```
instead of 
```
Mon, 
Aug 8:
```

---

2) **Where/how can I remove the extra newline between the different lines?**

Currently, the site renders an extra newline between the different items. For example, take a look at how it displays [Monday's tasks](https://ucsb-csw8.github.io/m22-ykk/calendar/#week-2) even though there are no spaces in the [file that generated it](https://github.com/ucsb-csw8/m22-ykk/blob/master/_modules/week-02.md). 

---

3) **Is there a way to prevent the text in the "middle column" from turning bold when there is an item in the "rightmost column"?**

The time for the class during which the Quiz occurs is displayed in bold, whereas the other two classes are not bolded.

---

I don't actually know Jekyll, which is a testament to this template and its documentation (and the maintainer @kevinlin1!) making it is so easy to pattern-match and get things running.

Thank you in advance for your help!
~Kate

**Comment by kevinlin1** [comment, marked as answer]
Hi Kate,

Great questions! Before getting started, it helps to review your browser's DevTools. For example, if you use Chrome, you'll want to know how to [Open Chrome DevTools](https://developer.chrome.com/docs/devtools/open/) and use it to [View and change CSS](https://developer.chrome.com/docs/devtools/css/). You can then play around with adjusting the CSS styles of elements and debug the website appearance.

[Screen recording 2022-07-31 9.45.47 AM.webm](https://user-images.githubusercontent.com/1915715/182036758-9e077564-0fd2-444a-b176-eb4441fe1f0f.webm)

I recommend checking that your changes work across different display widths. Even though students most likely use laptops for their computing classwork, they might sometimes have the page side-by-side with another page so it's important to test for narrow widths too.

Remember that changes made in your browser's DevTools aren't saved to the website, so you'll need to copy them into the `scss` files. The specific change that I made in the screen recording needs to be edited in the `_sass/custom/module.scss` file. https://github.com/kevinlin1/just-the-class/blob/2729a42458ae8f9dc5242fd959cc17cade19baeb/_sass/custom/module.scss#L31-L33

Using this knowledge, here's how I'd address your questions:

1. See the screen recording above for a quick fix to give the date column more space relative to the content for that day. The original `1fr 7fr` gives 1/8 of the width to the date; the new `1fr 5fr` gives 1/6 of the width to the date.
2. I think this will be fixed once we address the first question! The extra space appears because the date column is wrapping onto an extra line, and the way the layout matches the date to the content in a grid-like manner.
3. Try removing or modifying the `font-weight` style for `main-content dt` in your DevTools. https://github.com/kevinlin1/just-the-class/blob/2729a42458ae8f9dc5242fd959cc17cade19baeb/_sass/custom/custom.scss#L75-L82
